lib/utils/createConfig.js

```diff
@@ -202,16 +154,12 @@ function createConfig(config, argv, { port }) {
   }
 
   if (argv.openPage) {
-    options.open = true;
     options.openPage = argv.openPage;
   }
 
-  if (typeof argv.open !== 'undefined') {
-    options.open = argv.open !== '' ? argv.open : true;
-  }
-
-  if (options.open && !options.openPage) {
-    options.openPage = '';
+  // eslint-disable-next-line no-undefined
```

Comment:
It seems unnecessary. Or if we need it, We'll be able move this rule to .eslintrc.

Reply:
It is necessary, but I moved to the top of the `createConfig` file for convenience in this file. Let's remove the `no-undefined` requirement in another PR.